scheduler/config-k8s.edn

```diff
@@ -26,7 +27,8 @@
                                                :command "exit 0"
                                                :pools #{"k8s-alpha" "k8s-gamma"}}
                               :node-blocklist-labels ["blocklist-nodes-with-this-label-key"]
-                              :use-google-service-account? false}}
+                              :use-google-service-account? false
+                              :verifying-ssl false}}
```

Comment:
Worth a comment here that this was "Used to work around auth problems with the k8s-java library >= version XX' (And in the other spots. Can you fix config-composite.edn in the same way too please)